.pre-commit-config.yaml

```diff
@@ -6,6 +6,6 @@ repos:
     -   id: check-merge-conflict
     -   id: check-symlinks
 -   repo: https://github.com/psf/black
-    rev: 19.3b0
+    rev: 22.3.0
```

Comment:
@aarchiba I had to make this change to get my pre-commit to work. I think this is the correct thing now that we've moved to the new black.